valhalla/proposals/migrations/0009_auto_20171003_2017.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
why two migrations?

Reply:
One was to add the field, the other adds a choice. We can squash them if it bothers you.